test/TestProjects/TypeSchemaMapping/Customized/CustomizedModel.cs

```diff
@@ -1,16 +1,73 @@
+using System.Text.Json;
 using Azure.Core;
+using NamespaceForEnums;
 
 namespace CustomNamespace
 {
     [CodeGenSchema("Model")]
+    [CodeGenSuppress("CustomizedModel(CustomFruitEnum, CustomDaysOfWeek)")]
```

Comment:
Would it make sense to take multiple parameters (representing the constructor parameters) instead of a single string representation? I'm a bit confused how this format of string is matched with a constructor in the MemberRemoverRewriter code.

Reply:
Yeah, let me try that.

The ToDisplayString call on this line is what's generating this string notation.
https://github.com/Azure/autorest.csharp/pull/587/files#diff-b9c3140b06337ba81fab9579487c05b4R99

Reply:
Hmm, and now that I think about it, are parameter names possible as well? What if types are the same for parameters, but names are different (e.g. we want to reorder a constructor's parameters that all happen to be the same - or mostly the same - types)?

Reply:
If the parameter types are the same it would be considered the same overload and handled by overriding logic.